tests/bluetooth/bsim/nrf_auraconfig/nrf_auraconfig_simulation.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# Copyright 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+if [ -z "$1" ]; then
+    echo "error: This script is not meant to be called directly"
```

Comment:
Then, please, add an `_` at the beggining of the script name
`_nrf_auraconfig_simulation.sh` so it will not be grabbed automatically and it is a bit more evident for users they are not meant to just call it.